src/program-editors.rst

```diff
@@ -39,4 +39,41 @@ TODO: work on version control systems
 Accessibility
 =============
 
-TODO: accessibility-related discussion
\ No newline at end of file
+Background
+----------------------------
+
+Accessibility in computing refers to the development and design of both software and hardware systems so as to be usable by people with disabilities. This includes screen readers for those with visual impairments and alternative input devices for those with motor impairments. Additionally, advances in computing technology over the previous decades have enabled richer communication for individuals with impairments. For example, augmentative and alternative communication (AAC) tools such as video calling or speech generating devices have improved for assisting individuals with impairments in producing or comprehending speech.
+ 
+Although personal computers were first sold to the general public in the 1970s, the process of making this technology accessible began to become more common around 1990. For example, the IBM Screen Reader for DOS was created in 1986 (initially only for employees) :cite:`cooke`, and in 1995, Windows 95 became the first Windows OS to have built in accessibility features, rather than as add-ons :cite:`history, microsoft`. It’s also important to note that the Americans with Disabilities Act (ADA) was passed in 1990 :cite:`ada`.
+ 
+Accessibility research also appeared around this time. Like many HCI topics, it first emerged from the field of human factors. In ‘Thirty-Something Million: Should They Be Exceptions?’, Vanderheiden highlights the need to consider individuals with impairments in product design and development :cite:`vanderheiden1990thirty`. Bergman and Johnson provide an overview of accessibility as it relates specifically to HCI :cite:`bergman1995towards`; Readers should refer to this work for an in-depth history.
+ 
+Modern work has continued to explore design philosophies of accessible computing. Notably, in ‘Ability-Based Design: Concept, Principles and Examples’, Wobbrock et al. propose a shift in the frame of reference for thinking about accessible interfaces from disability to ability, and encourages designers to think about and adapt to what users can do :cite:`wobbrock2011ability`.  The authors present seven principles for this design approach: ability (focus on what users can do), accountability (change systems, not users), adaptation (interfaces match to users’ abilities), transparency (interfaces give awareness of adaptations), performance (systems may model user performance), context (systems may model the effects of different contexts), and commodity (systems may use low-cost, available hardware).
```

Comment:
Same here + rather than the long list at the end of this paragraph, format it as a bulleted list